crates/forge/tests/integration/replace_bytecode.rs

```diff
@@ -102,3 +102,68 @@ fn libcall_in_cheated() {
 
     assert_passed(&result);
 }
+
+#[test]
+fn contract_not_deployed() {
+    let test = test_case!(indoc!(
+        r#"
+            use snforge_std::{declare, replace_bytecode, ReplaceBytecodeError};
+            use starknet::{ClassHash, contract_address_const};
+
+            #[test]
+            fn contract_not_deployed() {
+                let arbitrary_class_hash: ClassHash = 0x5.try_into().unwrap();
```

Comment:
Can we use a declared class hash there? This test relies on internal implementation details (i.e. the contract address existence being the first thing that is checked)

Reply:
[Use actual class hash not to rely on implementation](https://github.com/foundry-rs/starknet-foundry/pull/2106/commits/d733a02b8c1e7cd7b5458da3db6dbed1ae31ae07)